.github/pull_request_template.md

```diff
@@ -0,0 +1,7 @@
+### Context
+
+- Issue:
```

Comment:
Is `issue` for the ticket link? We normally link the ticket in Github and it connects them automatically. 

<img width="351" alt="image" src="https://github.com/user-attachments/assets/71367981-cc5d-425a-92fd-83b3360c2a0f" />

Or have I misunderstood what this is meant to be for? 🤔

Reply:
ah we've not been using that and adding it manually to the ticket, I'll remove issue and ensure the team start using that, good shout